libraries/Microsoft.BotKit/BotWorker.cs

```diff
@@ -33,9 +31,16 @@ public BotWorker(Botkit controller, BotWorkerConfiguration config)
             this.Config = config;
         }
 
+        /// <summary>
+        /// Gets the configuration for BotWorker.
+        /// </summary>
+        /// <value>The botkit Configuration.</value>
```

Comment:
why the mention to 'botkit'? Is that correct?

Reply:
Fixed. Thanks!